app/components/alerts_component/alerts_component.html.erb

```diff
@@ -1,63 +1,36 @@
-                          advice_page_path(school,
-                                           alert_content.alert.advice_page,
-                                           alert_content.alert.alert_type.advice_page_tab_for_link_to,
-                                           anchor: alert_content.alert.alert_type.link_to_section) %>
-            <% end %>
-          <% end %>
-          <%= alert_content.send(content_field) %>
-      <% end %>
-    <% end %>
-  <% else %>
-    <%# old style formatting used on advice pages will be removed %>
-    <% alerts.each do |alert_content| %>
-        <%= component 'notice', classes: 'mt-4', status: status_for_alert_colour(alert_content.colour) do |c| %>
-          <div class="row">
-            <% if show_icons %>
-              <div class="col-md-1 d-flex align-items-center">
-                <%= fa_icon(alert_icon(alert_content.alert, 'fa-3x')) %>
-              </div>
-            <% end %>
-            <div class="<%= show_icons ? 'col-md-11' : 'col-md-12' %>">
-              <%= alert_content.send(content_field) %>
-              <% if show_links && alert_content.alert.advice_page.present? %>
-                <div class="text-right">
-                  <%= link_to t('schools.show.find_out_more'),
-                              advice_page_path(school,
-                                               alert_content.alert.advice_page,
-                                               alert_content.alert.alert_type.advice_page_tab_for_link_to,
-                                               anchor: alert_content.alert.alert_type.link_to_section) %>
-                </div>
-              <% end %>
-            </div>
+  <% if title? || link? %>
+    <div class="row">
+      <div class="col d-flex flex-wrap justify-content-between align-items-center">
+        <% if title? %>
+          <h3><%= title %></h3>
+        <% end %>
+        <% if link? %>
+          <div>
+            <%= link %>
           </div>
         <% end %>
+      </div>
+    </div>
+  <% end %>
+  <% prompts.each do |prompt| %>
+    <%= prompt %>
+  <% end %>
+  <% alerts.each do |alert_content| %>
+    <%= component 'prompt',
+                  icon: alert_icon(alert_content.alert),
+                  fuel_type: alert_content.alert.alert_type.fuel_type,
+                  status: status_for_alert_colour(alert_content.colour) do |prompt| %>
+          <% if alert_content.alert.alert_type.find_out_more? %>
+          <% prompt.with_link do %>
+            <%= link_to t('schools.show.find_out_more'),
+                        advice_page_path(school,
+                                         alert_content.alert.advice_page,
+                                         alert_content.alert.alert_type.advice_page_tab_for_link_to,
+                                         anchor: alert_content.alert.alert_type.link_to_section) %>
+          <% end if show_links %>
+        <% end %>
+        <%= alert_content.send(content_field) %>
     <% end %>
```

Comment:
There's something odd going on with the indentation here (alert_content is too far out)?

Reply:
That's where erblint wants it to be so parameters of all the functions align